src/lib/yang/schema.lua

```diff
@@ -794,14 +794,15 @@ function resolve(schema, features)
          end
       end
       if node.body then
-         node.body = shallow_copy(node.body)
-         for k,v in pairs(node.body or {}) do
+         node.body = {}
```

Comment:
this won't work, you need to read node.body before you replace with an empty body :)